### PR TITLE
Mat 6401 no context error

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-6401](https://jira.cms.gov/browse/MAT-6401)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/editor/StatusHandler.tsx [hunk 1]
```diff
@@ -18,6 +18,9 @@ const generateMadieAlertWithContent = (
   const warnings = warningAnnotation?.map((el) => {
     return <li>{transformAnnotation(el)}</li>;
   });
+  if (type === "success" && errorAnnotation && errorAnnotation.length > 0) {
+    type = "warning";
+  }
```
Comment (adongare) [inline src/components/editMeasure/editor/StatusHandler.tsx:23]: I'm lil bit confused about this. why do we need to change this to warning?

Comment (sb-cecilialiu) [reply src/components/editMeasure/editor/StatusHandler.tsx:23]: @adongare So this is when the cql is saved successfully, but there is error like no context. I guess we don't have to change it to warning, but it seems odd to me when there is error with the cql.

Comment (adongare) [reply src/components/editMeasure/editor/StatusHandler.tsx:23]: yeah but then not having context makes whole cql invalid(as per this ticket) and if you go to population criteria or test case tell user that there is an error with cql. therefore I think this should remain an error type and not warning

Comment (adongare) [reply src/components/editMeasure/editor/StatusHandler.tsx:23]: okay, I checked lil bit more and its not consistent. I see error sometimes and success sometimes. maybe we can create new ticket and talk to POs. please ignore my previous messages.

**Review by adongare — APPROVED** [review]

**Pull request merged**